### PR TITLE
New Style: Gradient Light 

### PR DESCRIPTION
**Pull request by samiaab1990**
Hi there,
Wanted to add a new style: Gradient Light 
GradientLightCSSCodeChunk

### CODE REVIEW

**Comment by yyyc514** [comment]
Please rebase this onto the current master branch instead of your very old fork.

**Comment by samiaab1990** [comment]
> Please rebase this onto the current master branch instead of your very old fork.

done and sorry! I'm still new to this

File: CHANGES.md [hunk 1]
```diff
@@ -105,6 +110,10 @@ Parser Engine Changes:
 
 New languages:
 
+
+New styles:
```
Comment (yyyc514) [inline CHANGES.md:114]: Please remove. This is already in the changelog.

File: CHANGES.md [hunk 2]
```diff
@@ -295,13 +305,14 @@ Language Improvements:
...
 
+
 ## Version 9.16.2
 
 New languages:
   none.
 
 New styles:
-  none.
+  -*Gradient Dark* by [Samia Ali][]
```
Comment (yyyc514) [inline CHANGES.md:315]: Please remove. This is already in the changelog.

File: src/styles/gradient-dark.css [hunk 3]
```diff
@@ -72,6 +72,7 @@ color:#e7e4eb;
 
 .hljs-attribute,
 .hljs-addition
+
```
Comment (yyyc514) [inline src/styles/gradient-dark.css:75]: Please remove this change.

**Comment by yyyc514** [comment]
No need for apology.  Better, but this still has a bunch of extra cruft.  All this should include:

- The new file
- A single line addition to the change log (or two lines, one for the footnote link)

**Comment by samiaab1990** [comment]
I did a hard reset and added the file again

**Comment by samiaab1990** [comment]
> I did a hard reset and added the file again

and theme to changelog and footnote (forgot). Thank you!

**Comment by yyyc514** [comment]
> I did a hard reset and added the file again

If you plan to contribute more in the future it's probably easier to never let your master get off our master and instead make a new branch anytime you want to make a PR. :-)

This is much cleaner, thanks!

File: CHANGES.md [hunk 4]
```diff
@@ -1,3 +1,11 @@
+## Version 10.1.2
```
Comment (yyyc514) [inline CHANGES.md:1]:
```suggestion
## Version 10.2.0
```

**Comment by yyyc514** [comment]
@samiaab1990 Thanks again!

**Pull request merged**